sct_docker.py

```diff
@@ -69,6 +69,10 @@ def generate(distro="debian:7", version="3.1.1", commands=None, name=None,
 RUN echo 'debconf debconf/frontend select Noninteractive' | debconf-set-selections
 RUN apt-get update
 RUN apt-get install -y curl
+
+# For Vim
+RUN apt-get update
```

Comment:
duplicate

Reply:
Yes, actually it didn't work without it. 
However, I am testing with `RUN apt-get update --fix-missing` which seems to work 
I will commit if it allows the generation of the docker container.